docs/resources/ocean_aks_vng_2.0.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: ocean_aks_np_vng"
```

Comment:
expand vng

Reply:
updated to ocean_aks_np_virtual_node_group